src/main/java/gmbs/Checker.java

```diff
@@ -0,0 +1,32 @@
+package gmbs;
+
+import static java.lang.Character.isDigit;
+
+// 수가 조건에 맞는 지 판별해주는 함수를 모아둔 클래스
+public class Checker {
+
+    // 숫자가 아닌 게 있는 지 확인해주는 함수
+    private boolean checkNotNumber(String number) {
+        if (isDigit(number.charAt(0)) && isDigit(number.charAt(1)) && isDigit(number.charAt(2))) {
+            return false;
+        }
+
+        return true;
```

Comment:
```suggestion
        return !isDigit(number.charAt(0)) || !isDigit(number.charAt(1)) || !isDigit(number.charAt(2));
```
위와 같은 방식으로 조건문을 바로 return 할 수도 있어요~

Reply:
커밋할 때도 리뷰해주신 방식으로 return하는 걸 프로그램이 권고했었는데 제가 쓴 코드로 return 하는 게 더 보기 좋은 코드인 줄 알았습니다 ㅠ
Checker를 전폭 수정해서 위 부분은 없어졌지만 다음에 작성할 때 이런 상황이 발생한다면 리뷰해주신 방식으로 반환하겠습니다!